### PR TITLE
UiSettings wrapper

### PR DESCRIPTION
**Pull request by deextercz**
#8 
Hello,
I just created wrapper for UI Settings, but there are some things to discuss.
More info in pull request

### CODE REVIEW

File: Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.Android/MapRenderer.cs [hunk 1]
```diff
@@ -208,24 +211,66 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
...
+            }
+
+            if (e.PropertyName == UiSettings.HasTiltEnabledProperty.PropertyName)
+            {
+                NativeMap.UiSettings.TiltGesturesEnabled = Map.UiSettings.HasTiltEnabled;
+            }
+
+            if (e.PropertyName == UiSettings.HasZoomEnabledProperty.PropertyName)
+            {
+                //It would be better to separate these properties
```
Comment (deextercz) [inline Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.Android/MapRenderer.cs]: Separate these properties to Control and Gesture?

File: Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.UWP/MapRenderer.cs [hunk 2]
```diff
@@ -105,27 +198,43 @@ void OnCollectionChanged(object sender, NotifyCollectionChangedEventArgs e)
             {
                 case NotifyCollectionChangedAction.Add:
                     foreach (Pin pin in e.NewItems)
+                    {
                         LoadPin(pin);
+                    }
                     break;
                 case NotifyCollectionChangedAction.Move:
-                    // no matter
+                    foreach (Pin pin in e.NewItems)
```
Comment (deextercz) [inline Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.UWP/MapRenderer.cs]: Will be removed

File: Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.Android/MapRenderer.cs [hunk 3]
```diff
@@ -114,12 +114,15 @@ void OnMapReady(GoogleMap map)
                 map.SetOnCameraChangeListener(this);
                 map.SetOnMapClickListener(this);
                 map.SetOnMapLongClickListener(this);
-                map.UiSettings.MapToolbarEnabled = false;
-                map.UiSettings.ZoomControlsEnabled = Map.HasZoomEnabled;
-                map.UiSettings.ZoomGesturesEnabled = Map.HasZoomEnabled;
-                map.UiSettings.ScrollGesturesEnabled = Map.HasScrollEnabled;
-                map.MyLocationEnabled = map.UiSettings.MyLocationButtonEnabled = Map.IsShowingUser;
-                map.TrafficEnabled = Map.IsTrafficEnabled;
+                map.UiSettings.CompassEnabled = Map.UiSettings.IsCompassEnabled;
```
Comment (deextercz) [inline Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.Android/MapRenderer.cs]: Use methods instead

**Comment by amay077** [comment]
Hi, sorry I'm late.
I created branch [issue_8](https://github.com/amay077/Xamarin.Forms.GoogleMaps/tree/issue_8).
Please re-send PR to issue_8 branch.

Because I think It will be little big changes.
Probably I need to some implementing after your PR merged.

**Pull request closed without merging**